latest checker dataflow failing on try-finally

I get a crash when computing a dataflow graph for this code:
```java
    static Object tryFinally() {
        Object x = new Object();
        try {
            return new Object();
        } finally {
            if (x != null) {
                return x;
            }
        }
    }
```
Top of the stack:
```
 Caused by: java.lang.NoSuchFieldError: operator
  	at org.checkerframework.javacutil.trees.FullyTreeCopier.visitBinary(FullyTreeCopier.java:67)
  	at org.checkerframework.javacutil.trees.FullyTreeCopier.visitBinary(FullyTreeCopier.java:40)
  	at com.sun.tools.javac.tree.JCTree$JCBinary.accept(JCTree.java:1993)
  	at com.sun.tools.javac.tree.TreeCopier.copy(TreeCopier.java:61)
  	at org.checkerframework.javacutil.trees.FullyTreeCopier.copy(FullyTreeCopier.java:48)
  	at org.checkerframework.javacutil.trees.FullyTreeCopier.copy(FullyTreeCopier.java:40)
  	at com.sun.tools.javac.tree.TreeCopier.visitParenthesized(TreeCopier.java:333)
  	at com.sun.tools.javac.tree.TreeCopier.visitParenthesized(TreeCopier.java:45)
  	at com.sun.tools.javac.tree.JCTree$JCParens.accept(JCTree.java:1849)
  	at com.sun.tools.javac.tree.TreeCopier.copy(TreeCopier.java:61)
  	at org.checkerframework.javacutil.trees.FullyTreeCopier.copy(FullyTreeCopier.java:48)
  	at org.checkerframework.javacutil.trees.FullyTreeCopier.copy(FullyTreeCopier.java:40)
  	at com.sun.tools.javac.tree.TreeCopier.visitIf(TreeCopier.java:238)
  	at com.sun.tools.javac.tree.TreeCopier.visitIf(TreeCopier.java:45)
```

Might be a bad interaction with JDK 9 javac.  If we can't find an easy workaround, may have to revert back to 2.1.14.

--- a/README.md
+++ b/README.md
@@ -34,7 +34,7 @@ dependencies {
     compile "com.google.code.findbugs:jsr305:3.0.2"
 
     errorprone "com.google.errorprone:error_prone_core:2.1.1"
-    errorprone "org.checkerframework:dataflow:2.2.0"
+    errorprone "org.checkerframework:dataflow:2.1.14"
 
 }
 

--- a/gradle/dependencies.gradle
+++ b/gradle/dependencies.gradle
@@ -15,7 +15,7 @@
  */
 
 def versions = [
-    checkerFramework : "2.2.0",
+    checkerFramework : "2.1.14",
     errorProne       : "2.1.1",
 ]
 

--- a/nullaway/src/main/java/com/uber/nullaway/dataflow/DataFlow.java
+++ b/nullaway/src/main/java/com/uber/nullaway/dataflow/DataFlow.java
@@ -94,14 +94,16 @@ public final class DataFlow {
                                         LambdaExpressionTree lambdaExpressionTree =
                                                 (LambdaExpressionTree) methodPath.getLeaf();
                                         ast = new UnderlyingAST.CFGLambda(lambdaExpressionTree);
-                                        bodyPath = new TreePath(methodPath, lambdaExpressionTree.getBody());
+                                        return CFGBuilder.build(
+                                                methodPath.getCompilationUnit(), env, ast, false, false);
                                     } else {
                                         // must be a method per findEnclosingMethodOrLambda
                                         MethodTree method = (MethodTree) methodPath.getLeaf();
                                         ast = new UnderlyingAST.CFGMethod(method, /*classTree*/ null);
                                         bodyPath = new TreePath(methodPath, method.getBody());
+                                        return CFGBuilder.build(
+                                                bodyPath, env, ast, false, false);
                                     }
-                                    return CFGBuilder.build(bodyPath, env, ast, false, false);
 
                                 }
                             });

--- a/nullaway/src/test/resources/com/uber/nullaway/testdata/NullAwayNegativeCases.java
+++ b/nullaway/src/test/resources/com/uber/nullaway/testdata/NullAwayNegativeCases.java
@@ -496,6 +496,18 @@ public class NullAwayNegativeCases {
         }
     }
 
+    static Object tryFinally() {
+        Object x = new Object();
+        try {
+            return new Object();
+        } finally {
+            if (x != null) {
+                return x;
+            }
+        }
+    }
+
+
     static class ClassWithConstructor {
 
         @Nullable Object f;

